.wp-env.json

```diff
@@ -0,0 +1,4 @@
+{
+  "core": "WordPress/WordPress#master",
+  "plugins": [ "." ]
```

Comment:
Does GP have to be listed here or does the 'npm install' pick up the checked out directory automatically?

Reply:
Good question. The dot means that the current directory will be symlinked to the containers to avoid checking out the same plugin again.
See https://developer.wordpress.org/block-editor/packages/packages-env/#wp-env-json for other allowed values.

When used outside of this repo you should be able to do something like this too:
```json
{
  "core": "WordPress/WordPress#master",
  "plugins": [ "GlotPress/GlotPress-WP#develop" ]
}
```

This will check out the git repo of WordPress and GlotPress.